test/libsolidity/smtCheckerTests/bmc_coverage/math.sol

```diff
@@ -22,4 +22,4 @@ contract C {
 // Warning 4144: (217-222): BMC: Underflow (resulting value less than 0) happens here.
 // Warning 2661: (293-298): BMC: Overflow (resulting value larger than 2**256 - 1) happens here.
 // Warning 3046: (369-374): BMC: Division by zero happens here.
-// Warning 6084: (68-73): BMC: Underflow (resulting value less than 0) happens here.
+// Warning 4144: (68-73): BMC: Underflow (resulting value less than 0) happens here.
```

Comment:
There seems to be a special warning code when an issue is detected, but the call stack is empty.
@leonardoalt, do you know why this is the case? With this change, if an issue is detected in the implicit constructor, the error code reported will be the same as in other places. (Compare the two underflow reports: before they had different error codes, now they have the same.)

Reply:
And this removes the only occurrence of this error code in the test suite, so the `chk_errorcodes` check now fails. What should be done with this error code?

Reply:
I think we can remove the error code and the if/else where it checks that the callstack is empty or not, and assert that the callstack is not empty

Reply:
Such assert is present in the method `SMTEncoder::callStackMessage` which is the only place where the call stack is actually used, but I have added one also at the beginning, just in case.